### PR TITLE
Depend on CDAP 6

### PR DESCRIPTION
**Pull request by rohitsinha54**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ These files can be used to deploy your plugins.
 
 UI Integration
 --------------
-The Cask Hydrator UI displays each plugin property as a simple textbox. To customize how the plugin properties
+The CDAP Studio UI displays each plugin property as a simple textbox. To customize how the plugin properties
```
Comment (anwar6953) [inline README.md]: Pipeline Studio

**Review by anwar6953 — APPROVED** [review]

File: src/main/java/io/cdap/plugin/FastFilter.java [hunk 2]
```diff
@@ -14,22 +14,21 @@
...
-import co.cask.cdap.api.annotation.Name;
-import co.cask.cdap.api.annotation.Plugin;
-import co.cask.cdap.api.data.format.StructuredRecord;
-import co.cask.cdap.api.data.schema.Schema;
-import co.cask.cdap.api.plugin.PluginConfig;
-import co.cask.cdap.etl.api.Emitter;
-import co.cask.cdap.etl.api.PipelineConfigurer;
-import co.cask.cdap.etl.api.Transform;
-import co.cask.cdap.etl.api.TransformContext;
+package io.cdap.plugin;
```
Comment (anew) [inline src/main/java/io/cdap/plugin/FastFilter.java]: according to the guidelines, this package should be io.cdap.plugin.fastfilter

**Review by anew — CHANGES_REQUESTED** [review]
Can you also change it to use the cdap-maven-plugin?

File: pom.xml [hunk 3]
```diff
@@ -84,143 +84,83 @@
...
-      </plugins>
-    </pluginManagement>
-
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.1</version>
+        <configuration>
+          <source>1.7</source>
```
Comment (anwar6953) [inline pom.xml:91]: 1.8?

File: pom.xml [hunk 4]
```diff
@@ -84,143 +84,83 @@
...
       </plugin>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-antrun-plugin</artifactId>
+        <groupId>io.cdap</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.1.0</version>
+        <configuration>
+          <cdapArtifacts>
+            <parent>system:cdap-data-batch[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```
Comment (anwar6953) [inline pom.xml]: We don’t need all 4 parents right?

File: pom.xml (hunk 3 above)
Comment (rohitsinha54) [reply pom.xml:91]: Do we need to do it if we are not using any 1.8 features?

**Review by anwar6953 — APPROVED** [review]

**Pull request merged**